### DISCUSSION TITLE
How to play audio from your PC to Jamulus (using JACK) - Knowledge Base Discussion

### DISCUSSION THREAD

**Discussion by ann0see** [jamulussoftware/jamulus#3088] [Community Knowledgebase]
This is the discussion for the article [How to play audio from your PC to Jamulus (using JACK)](https://jamulus.io/kb/2022/01/26/Routing-JACK-Windows.html) feel free to comment and talk about improving the article here.

**Comment by jujudusud** [comment]
Hi here,

Good kb article.

Is it possible to rename with Windows in its name ?

**Reply by ann0see** [reply]
Thanks! At best we'd add a section about how it would look like on Linux too. 
Edit: The page URL says Windows, so probably we should stay focused on this OS.

**Reply by trebmuh** [reply]
I'd rename it to "How to play audio from your PC to Jamulus (using JACK on Windows)"

**Reply by ann0see** [reply]
Thanks for the suggestion. @jujudusud feel free to update the page title (on the next-release branch)

**Comment by pljones** [comment]
I'd forgotten about this!  Just what I needed.

In my set up, I mix in Reaper with ReaRoute -- Jack has ReaRoute as its ASIO output.  This means I just continue using the ASIO Jamulus client, also linked through Reaper with ReaRoute.  ASIO Bridge and Jack just bring system sound into Reaper.  Very neat.

**Reply by pljones** [reply]
In fact... now I've written that down, I see I can do without Jack entirely.  Just feed ASIO Bridge to ReaRoute directly.
